docs/advanced-usage/using-proxies.mdx

```diff
@@ -102,6 +102,43 @@ When using a proxy, all requests to the Frontend API will be made through your d
         ```
       </CodeBlockTabs>
     </Tab>
+
+    <Tab>
+      ```ts {{ prettier: false, filename: 'middleware.ts' }}
+        import { NextResponse } from "next/server";
+        import { clerkMiddleware } from "@clerk/nextjs/server";
+
+        export default clerkMiddleware((auth, request) => {
+          if (request.url.match("__clerk")) {
+            const proxyHeaders = new Headers(request.headers);
+            proxyHeaders.set("Clerk-Proxy-Url", "https://app.dev/__clerk");
```

Comment:
Can the `process.env. NEXT_PUBLIC_CLERK_PROXY_URL` be reused here?

Reply:
That's a great call, I'll update this now!